roles/flannel/meta/main.yml

```diff
@@ -14,3 +14,4 @@ galaxy_info:
   - system
 dependencies:
 - { role: openshift_facts }
+- { role: etcd_client_certificates, etcd_generated_certs_dir: /etc/etcd/generated_certs, etcd_cert_prefix: node.etcd- }
```

Comment:
Nah, I copied this out right from where it was defined in the playbook- I will change to flannel. The destination directory is configurable but is set at the playbook level. I assumed these vars would be static.